web/crux/src/app/deploy/deploy.mapper.ts

```diff
@@ -91,7 +89,7 @@ export class DeployMapper {
       config: {
         capabilities: (instance.config?.capabilities as UniqueKeyValue[]) ?? [],
         environment: (instance.config?.environment as UniqueKeyValue[]) ?? [],
-        config: this.imageMapper.explicitConfigToGrpc(config.config as ExplicitContainerConfigData),
+        config: config.config as JsonObject,
```

Comment:
i can't se why

Reply:
config.config is Prisma.JsonValue and I preffered JsonObject over any in this case to be respectful.

Reply:
but this maps to grpc isn't it?